db/migrate/20200908121903_add_branch_to_pull_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddBranchToPullRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pull_requests, :branch, :text
```

Comment:
Just curious, why `text` instead of `string`?

Reply:
Nothing in particular. Would you recommend using string?

Reply:
I think they're effectively pretty similar, so I'll use whatever it's being used for this kind of values ✨

Reply:
They are both being used... although `string` seems to be more popular :P
I'll see if I can quickly change it